config/routes.rb

```diff
@@ -51,7 +51,7 @@
 
       resources :evidence, concerns: :multiple_destroy, controller: 'issues/evidence', only: [:index, :new]
       resources :nodes, only: [:show], controller: 'issues/nodes'
-      resources :revisions, only: [:index, :show]
+      resources :revisions, only: [:index, :show] 
```

Comment:
extra space here, L78, L82

@aapomm this could have been a peer-review suggestion?